TESTING.md

```diff
@@ -43,6 +43,9 @@ To run specific integration tests, pass the path to the test:
 `yarn test:jest_integration [test path]`
 
 ### Functional tests
+
+Functional testing in OpenSearch Dashboards is migrating to the [opensearch-dashboards-functional-test](https://github.com/opensearch-project/opensearch-dashboards-functional-test) repository. All new functional tests should be written there. When modifying a file that affects an exisitng functional test, the old test should be migrated to the new repository. The rest of this section outlines how to run the existing functional tests in the repository.
```

Comment:
typo: existing

Reply:
> the old test should be migrated to the new repository.

It would be nice to also have some pointers on how to do this...

Reply:
@joshuarrrr We dont have a process to defined for this yet since we havent done a migration. The summary view PR can be a good example where we set precedent and come up with good examples. We can update this text once we do. I wanted to add this message here now because this is where i looked when I was writing tests and didnt see any mention of the FTR repo and its use.

@abbyhu2000 thanks, will make the change